src/styles/reset.scss

```
@@ -0,0 +1,9 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+body,
+p,
+h2 {
+  margin: 0;
+}
```

add 1 empty line after your code
```suggestion
}

```